src/test/java/com/napier/sem/recordClasses_UnitTests.java

```diff
@@ -0,0 +1,83 @@
+package com.napier.sem;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class recordClasses_UnitTests
+{
+    @Test
+    public void capitalCity_ensureGettersReturnsCorrectFields()
```

Comment:
these names don't mention tostring, which they also test. something to be fixed for tomorrow